app/templates/language-wikis-index.hbs

```diff
@@ -0,0 +1 @@
+{{language-wikis-index}}
```

Comment:
can we render this directly here, instead of the component wrapper? 
It'll be easier to follow